### DISCUSSION TITLE
hyperkey sublayer w + d/b/n/m not usable

### DISCUSSION THREAD

**Discussion by LK2408** [mxstbr/karabiner#35] [Q&A]
super useful command but I can't use them anywhere. I also cloned the repo again but yeah they are not working anywhere for me. Only thing b / m do at the moment is indent and not indent. Any advice how to use them?

**Comment by mxstbr** [comment]
`d` doesn't work for me either, I switched that to `y` and `o`. Will remove `d`. 

`b` and `m` work for me to go back and forth in Chrome.

`n` works for me to switch between windows of any app.

**Reply by mxstbr** [reply]
Removed `d` in 57fd6db!

**Reply by LK2408** [reply]
Is `n` similiar to the macos setting where you just go to the next window/next active window? It is not working to me so I set the macos next window setting to `right_command + grave_accent_and_tilde`?

**Reply by mxstbr** [reply]
Yep, that's exactly what it's meant to trigger 😊
